### ISSUE TITLE
Mautic 2.15.3 No longer Pushes Leads to SuiteCRM

### ISSUE THREAD

**Issue by pstevens71** [mautic/mautic#8046]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
The more details the better...
Previously, pushing leads from Mautic to SuiteCRM was working without issue.  Just updated to latest version 2.15.3 and now it creates a contact in SuiteCRM where all the fields are blank.  I've checked the error log and nothing is there.   When I push a lead on web form fill this happens.  I created a campaign to push the lead one off, and it does not succeed and says "generic error" in the contact record.

Note:  I have two different installations on two different servers with the same issue.

| Q   | A
| --- | ---
| Mautic version | 2.15.3
| PHP version | 7.1
| Browser | Chrome

### Steps to reproduce
1.  Create a webform that on fill pushes lead to SuiteCRM
2.  Fill in form
3. Lead is added to Mautic no problem, but pushes a blank record to SuiteCRM

(note: the SuiteCRM plugin was previously configured and working, double checked and all the right fields are still mapped).
 
### Log errors
No log errors.

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

**Comment by pstevens71** [comment]
After some trial and error, managed to get it working again.  Had to go into the plugin and "re-authorize app"  After re-authorizing the app the leads push to SuiteCRM again.  Hope this saves someone else some time troubleshooting.

**Comment by npracht** [comment]
It seems duplicate of https://github.com/mautic/mautic/issues/8040

**Comment by pstevens71** [comment]
Yeah it is a duplicate of #8040.  While re-authorizing solved some problems it did not fix the issue entirely.  Leads can't be pushed by campaign anymore and only if a lead is added through a Mautic form is it pushed via action to the integration.  I've updated my testing and trials on #8040.  I changed the suitecrm integration file back to the the 2.15.1 version I had and it works again.  Must be the changes in either .2 or .3 that caused the issue.
